docs/domains/Domain.md

```diff
@@ -34,6 +34,7 @@ The specification of the operation of the WebLogic domain. Required.
 | `logHome` | string | The directory in a server's container in which to store the domain, Node Manager, server logs, server *.out, introspector .out, and optionally HTTP access log files if `httpAccessLogInLogHome` is true. Ignored if `logHomeEnabled` is false. |
 | `logHomeEnabled` | Boolean | Specifies whether the log home folder is enabled. Defaults to true if `domainHomeSourceType` is PersistentVolume; false, otherwise. |
 | `managedServers` | array of [Managed Server](#managed-server) | Lifecycle options for individual Managed Servers, including Java options, environment variables, additional Pod content, and the ability to explicitly start, stop, or restart a named server instance. The `serverName` field of each entry must match a Managed Server that already exists in the WebLogic domain configuration or that matches a dynamic cluster member based on the server template. |
+| `maxClusterConcurrentShutdown` | number | The default maximum WebLogic Server instances that a cluster will shutdown in parallel when it is being partially shutdown by lowering its replica count. You can override this default on a per cluster basis by setting the cluster's `maxConcurrentShutdown` attribute. A value of 0 means there is no limit. Defaults to 1 |
```

Comment:
default maximum WebLogic Server instances -> default maximum number of WebLogic Server instances
shutdown -> shut down (globally)
Defaults to 1-> Defaults to 1.